nri_device_injector/nri_device_injector_test.go

```diff
@@ -15,12 +15,81 @@
 package main
 
 import (
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"golang.org/x/sys/unix"
 )
 
-func TestParseDevices(t *testing.T) {
+func TestToNRIDevice(t *testing.T) {
+	tempDir, err := os.MkdirTemp("", "device-test")
```

Comment:
You may want to skip this except when explicitly running root-requiring tests.  The pattern we use for this in containerd is https://github.com/containerd/containerd/blob/d9a58a892b77f292b842b849f059d8d8e8972b4a/pkg/testutil/helpers_unix.go#L29-L36

Reply:
I see, I'll try remove using the secret then.

Reply:
Done, the test is skipped if uid isn't 0.

Reply:
That's probably fine; there's a risk that someone who doesn't know that a test requires UID 0 will just miss running them, but since we have this going through CI now I think it's okay.